iamproperty/iamproperty.github.io
# Ds 73/modals

## Summary of Changes:
1. CSS
2. Vue component

## Rationale
Create a tabs component to hide away secondary information so that it can be viewed upon request.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/modal

## Ticket(s)
DS-73

## Notes
